clusterctl/clusterdeployer/clusterdeployer.go

```diff
@@ -118,69 +118,69 @@ func (d *ClusterDeployer) Create(cluster *clusterv1.Cluster, machines []*cluster
 		return fmt.Errorf("unable to seperate master machines from node machines: %v", err)
 	}
 
-	glog.Info("Creating external cluster")
-	externalClient, cleanupExternalCluster, err := d.createExternalCluster()
-	defer cleanupExternalCluster()
+	glog.Info("Creating bootstrap client")
```

Comment:
I'm torn about saying client here vs. cluster. In some cases, it will just be a client (to an existing cluster) but the default (at least today) is to actually create a cluster, in which case saying it this way will be confusing to users.